src/Arcus.Templates.WebApi/Serialization/HealthReportJson.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using GuardNet;
+
+// ReSharper disable once CheckNamespace
+namespace Microsoft.Extensions.Diagnostics.HealthChecks
```

Comment:
IMO, the class in this namespace is an API model.   Although it is not a 'business API model' but more an infrastructural model, I think I'd put it in a 'Models' namespace.

Reply:
Now it's in the serialization namespace as it's for serialization, a model namespace would be more for a business model, no? But, ok, we can move it there.

Reply:
Done in https://github.com/arcus-azure/arcus.templates/pull/589/commits/7da2befc3a9356df33cd00e8a891b83ff54b70d6

Reply:
Though decision :) 
I'd consider that types in a `Serialization` namespace are more 'infrastructural' types that you need to perform serialization.